x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_columns.tsx

```diff
@@ -273,9 +332,148 @@ export const useColumns = (
         sortable: true,
         valign: 'middle',
       },
+      ['Baseline rate']: {
```

Comment:
Can the `Baseline rate` column be made sortable?

Reply:
Updated in ff861eb3f280d3812b7536fcbabcd06eced039cc